lib/menu.rb

```diff
@@ -0,0 +1,31 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = import_dishes
+  end
+
+  def dishes
+     @dishes.clone
+  end
+
+  private
+
+  def import_dishes
+    full_menu = {
+      :vegetable_soup => 5.5,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)